test/ctr.bats

```diff
@@ -212,7 +212,7 @@ function teardown() {
 	run crictl start "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run crictl rm "$ctr_id"
```

Comment:
Sorry if a dupe comment, having fun network issues.  Although very minimal, I'd really rather not see test changes in a version bump PR unless they would break due to the bump.  I think these should be in a separate PR.

Reply:
I got this change from @saschagrunert's PR. Thought it was needed, though never checked. Let's find out :smile:

Reply:
yes, it appears without the change the tests break

Reply:
Thanks for checking @haircommander, I was fine with them in, more of a reminder for the future.  But saying that, I'm puzzled that it's required, I would not have guessed that.